test_files/comments.ts

```diff
@@ -1,8 +1,19 @@
-class A {
+class Comments {
   /** @export */
-  foo: string;
+  export1: string;
+  /// @export
+  export2: string;
   /* non js-doc comment */
-  bar: number;
+  nodoc1: number;
   // non js-doc comment
-  buz: number;
+  nodoc2: number;
+  /// inline jsdoc comment without type annotation
```

Comment:
Aren't you worried that this would change the meaning of (arguably invalid) // @export (something that closure would ignore AFAIK) into valid /** @export @type */ . Feels like we would promote wrong style.